packages/core/src/flow/chain.ts

```diff
@@ -25,7 +25,7 @@ decorate(chain, threadable());
  * functions. For example:
  * ```ts
  * yield* all(
- *   rect.corenerRadius(20, 3),
+ *   rect.cornerRadius(20, 3),
```

Comment:
This is actually outdated. It should be simply:
```suggestion
 *   rect.radius(20, 3),
```